prime-router/build.gradle.kts

```diff
@@ -742,9 +742,10 @@ dependencies {
     implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:$jacksonVersion")
     implementation("com.fasterxml.jackson.core:jackson-databind:$jacksonVersion")
     implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:$jacksonVersion")
-    implementation("com.github.javafaker:javafaker:1.0.2")
-    // Pin snakeyaml since it is getting included regardless of exclude attempts
-    implementation("org.yaml:snakeyaml:1.33")
+    implementation("com.github.javafaker:javafaker:1.0.2") {
+        exclude(group = "org.yaml", module = "snakeyaml")
```

Comment:
javafaker uses: <classifier>android</classifier>
which does not exist for snakeyaml:2.0